lib/log-parsers.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const jmespath = require('jmespath')
+
+const {
+  isObject
+} = require('./utils')
+
+const bourne = require('@hapi/bourne')
+const jsonParser = input => {
+  try {
+    return { value: bourne.parse(input, { protoAction: 'remove' }) }
+  } catch (err) {
+    return { err }
+  }
+}
+
+const logParsers = [
+  (input, { EOL }) => {
```

Comment:
Again, this is just a big blob of anonymous functions without any context. It will be very difficult to maintain.

Reply:
Agreed, again. The update I just pushed improves both parsers and builders.